core/src/main/scala/org/apache/spark/ui/jobs/AllJobsPage.scala

```diff
@@ -541,20 +541,23 @@ private[ui] class JobPagedTable(
 
   override def headers: Seq[Node] = {
     // Information for each header: title, cssClass, and sortable
-    val jobHeadersAndCssClasses: Seq[(String, String, Boolean)] =
+    val jobHeadersAndCssClasses: Seq[(String, String, Boolean, String)] =
       Seq(
-        (jobIdTitle, "", true),
-        ("Description", "", true), ("Submitted", "", true), ("Duration", "", true),
-        ("Stages: Succeeded/Total", "", false),
-        ("Tasks (for all stages): Succeeded/Total", "", false)
+        (jobIdTitle, "", true, ""),
+        ("Description", "", true, ""),
+        ("Submitted", "", true, ""),
+        ("Duration", "", true, "Elapsed time since the job was submitted to the spark" +
```

Comment:
Not sure about 'to the spark application' - it's the app that is submitted to Spark. Just remove it?
till -> until
-> until completion of all its stages